draft-ietf-tls-esni.md

```diff
@@ -210,13 +219,12 @@ keys
 : The list of keys which can be used by the client to encrypt the SNI.
 Every key being listed MUST belong to a different group.
 
-padded_length
-The length to pad the ServerNameList value to prior to encryption.
-This value SHOULD be set to the largest ServerNameList the server
-expects to support rounded up the nearest multiple of 16. If the
-server supports arbitrary wildcard names, it SHOULD set this value to
-260. Clients SHOULD reject ESNIConfig as invalid if padded_length is
-greater than 260.
+maximum_name_length
+: the largest name the server expects to support
+rounded up the nearest multiple of 16. If the server supports
+arbitrary wildcard names, it SHOULD set this value to
+256. Clients SHOULD reject ESNIConfig as invalid if maximum_name_length is
+greater than 256.
```

Comment:
"reject an ESNIConfig"?

I'm still not clear why there's server-controlled padding here. I can see using this figure as a hint, though.

Reply:
Just to add to the seemingly endless traffic identification via packet size: https://isc.sans.edu/diary/25616